clearwater-cluster-manager.root/usr/share/clearwater/cluster_manager/scripts/load_from_chronos_cluster

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# @file load_from_chronos_cluster
+#
+# Project Clearwater - IMS in the Cloud
+# Copyright (C) 2015  Metaswitch Networks Ltd
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or (at your
+# option) any later version, along with the "Special Exception" for use of
+# the program along with SSL, set forth below. This program is distributed
+# in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details. You should have received a copy of the GNU General Public
+# License along with this program.  If not, see
+# <http://www.gnu.org/licenses/>.
+#
+# The author can be reached by email at clearwater@metaswitch.com or by
+# post at Metaswitch Networks Ltd, 100 Church St, Enfield EN2 6BQ, UK
+#
+# Special Exception
+# Metaswitch Networks Ltd  grants you permission to copy, modify,
+# propagate, and distribute a work formed by combining OpenSSL with The
+# Software, or a work derivative of such a combination, even if such
+# copying, modification, propagation, or distribution would otherwise
+# violate the terms of the GPL. You must comply with the GPL in all
+# respects for all of the code used other than OpenSSL.
+# "OpenSSL" means OpenSSL toolkit software distributed by the OpenSSL
+# Project and licensed under the OpenSSL Licenses, or a work based on such
+# software and licensed under the OpenSSL Licenses.
+# "OpenSSL Licenses" means the OpenSSL License and Original SSLeay License
+# under which the OpenSSL Project distributes the OpenSSL toolkit software,
+# as those licenses appear in the file LICENSE-OPENSSL.
+
+set -ue
+
+. /etc/clearwater/config
+
+if [ $# -ne 1 ]
+then
+  echo "Usage: load_from_chronos_cluster [node_type]"
+  exit 1
+fi
+
+node_type=$1
+
+/usr/share/clearwater/clearwater-cluster-manager/env/bin/python /usr/share/clearwater/cluster_manager/scripts/dump_etcd_state.py $local_ip $node_type
```

Comment:
These should both be `/usr/share/clearwater/clearwater-cluster-manager/...` - one of my directory names is wrong. I'll fix up in my own scripts/move that directory/check to make sure `/usr/share/clearwater/cluster_manager/` isn't referenced anywhere else.